Makefile

```diff
@@ -31,6 +33,12 @@ ifeq ($(PARENT_BRANCH_SUFFIX), main)
 CERT_MANAGER_OPERATOR_BRANCH = master
 endif
 
+ISTIO_PARENT_BRANCH = $(strip $(shell git branch --show-current))
+
+ISTIO_BRANCH_SUFFIX = $(if $(filter release-%, $(ISTIO_PARENT_BRANCH)),$(strip $(shell echo $(ISTIO_PARENT_BRANCH) | cut -d'-' -f2)),main)
+
+ISTIO_CSR_BRANCH ?= $(if $(filter release-%, $(ISTIO_PARENT_BRANCH)),release-$(ISTIO_BRANCH_SUFFIX),main)
+
```

Comment:
istio-csr repository too will have release branches, ex `release-1.15`, we won't be needing the changes related to branch name parsing.